app/scripts/views/mixins/signed-out-notification-mixin.js

```diff
@@ -34,11 +33,9 @@ define(function (require, exports, module) {
     },
 
     navigateToSignIn () {
-      this.navigate('signin', {
-        success: BaseView.t('Signed out successfully')
-      }, {
-        clearQueryParams: true
-      });
+      // Navigate via the back-end in order to regenerate
+      // flow id and flow begin time for the next session.
+      this.window.location.href = '/signin';
```

Comment:
Ah, gotcha - maybe build a link similar to how we do in [connect-another-device.js](https://github.com/mozilla/fxa-content-server/blob/6fa4336c19b255fff4657045a12b85eb9f8270b1/app/scripts/views/connect_another_device.js#L226:L247)